src/components/pages/ApartmentEditPage.jsx

```diff
@@ -1,43 +1,67 @@
-import React, { useEffect } from "react";
-import { useParams, useLocation, matchPath } from "react-router-dom";
+import React from "react";
+import { matchPath, Redirect } from "react-router-dom";
 import withApartments from "components/apartments/withApartments";
 import PageCard from "components/atoms/PageCard";
+import compose from "just-compose";
 import { Alert } from "reactstrap";
-import { useTranslation } from "react-i18next";
+import { withTranslation } from "react-i18next";
+import memoize from "lodash-es/memoize";
+import { withToastManager } from "react-toast-notifications";
 
 import InProgress from "components/atoms/InProgress";
 import PageErrorMessage from "components/atoms/PageErrorMessage";
 import PageNavigationApartmentList from "components/atoms/PageNavHome";
 import ApartmentForm from "components/apartment/ApartmentForm";
 import Routes from "constants/Routes";
-import memoize from "lodash-es/memoize";
 import { ApartmentFormFields } from "components/apartment/ApartmentFormFields";
 import Apartment from "models/Apartment";
+import { classToPlain } from "serializers/Serializer";
+import { Toast } from "components/atoms/Toast";
+import { updateApartment } from "services/Api";
+
+const EDITING = 0;
+const SENDING = 1;
+const SENT = 2;
```

Comment:
Lets keep these constants somewhere outside this file, so we have a shared point of truth :)